src/pages/languages.tsx

```diff
@@ -0,0 +1,200 @@
+import { Box, Flex, IconButton, LinkBox, LinkOverlay } from "@chakra-ui/react"
+import React, { useState } from "react"
+import { MdClose } from "react-icons/md"
+import { useTranslation } from "next-i18next"
+import { useRouter } from "next/router"
+import { GetStaticProps } from "next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+
+import InlineLink, { BaseLink } from "../components/Link"
+import Input from "../components/Input"
+import PageMetadata from "../components/PageMetadata"
+import Text from "../components/OldText"
+import OldHeading from "../components/OldHeading"
+
+import { getRequiredNamespacesForPath } from "@/lib/utils/translations"
+import { languageMetadata } from "@/lib/utils/languages"
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+
+import { TranslationKey } from "@/lib/types"
+import { Language } from "@/lib/interfaces"
+
+export const getStaticProps: GetStaticProps = async (context) => {
```

Comment:
Lets use the `satisfies` form as we are doing in [the `slug` file](https://github.com/ethereum/ethereum-org-fork/blob/dev/src/pages/%5B...slug%5D.tsx#L171) for consistency and improved type checking.

```suggestion
export const getStaticProps = async (context) => {
```

Reply:
We could also declare the props being used here
```ts
type Props = SSRConfig & {
  lastDeployDate: string
}

// and then
... satisfies GetStaticProps<Props>
```

Reply:
Fixed